build.gradle.kts

```diff
@@ -46,9 +46,10 @@ dependencies {
   implementation("org.apache.commons:commons-csv:1.11.0")
 
   runtimeOnly("org.flywaydb:flyway-core")
+  runtimeOnly("org.flywaydb:flyway-database-postgresql:10.17.1")
```

Comment:
in visit scheduler and other repos we don't reference the version for these 2

  runtimeOnly("org.flywaydb:flyway-core")
  runtimeOnly("org.flywaydb:flyway-database-postgresql")

should we be doing this?

Reply:
good point, will remove the version.